### PR TITLE
Bug fix/8541 issue fixes

### PR DESCRIPTION
**Pull request by ravirk91**
Bug fix/8541  issue fixes

### CODE REVIEW

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066&bid=17709207)

File: Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs [hunk 1]
```diff
@@ -241,6 +242,11 @@ private void ReceiveCallback(IAsyncResult ar)
                 }
                 throw e;
             }
+            catch(OutOfMemoryException ex)
+            {
+                Console.WriteLine("OutOfMemoryException: ", ex.Message);
```
Comment (ambarkhan) [inline Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs:247]: Codacy Issue found: [The format string might be wrong, the following arguments are unused: 'ex.Message'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066)

Comment (ambarkhan) [inline Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs:247]: Codacy Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066&bid=17709207)

File: Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs [hunk 2]
```diff
@@ -241,6 +241,11 @@ private void ReceiveCallback(IAsyncResult ar)
...
```
Comment (ambarkhan) [inline Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs:246]: Codacy Issue found: [The format string might be wrong, the following arguments are unused: 'ex.Message'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066)

Comment (ambarkhan) [inline Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs:246]: Codacy Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5402066)

Comment (JinendraGhodke) [inline Ginger/GingerPluginCore/CommLib/GingerSocketInfo.cs:246]: Add Reporter.ToLog as well. Console will write only in the console

**Pull request merged**